components/Buttons/src/MDCFloatingButton.h

```diff
@@ -37,7 +37,15 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
   /**
    A 56-point circular button surrounding a 36-point square icon or short text.
    */
-  MDCFloatingButtonShapeLargeIcon = 2
+  MDCFloatingButtonShapeLargeIcon = 2,
```

Comment:
Let us acknowledge that:
enums are much more difficult to deprecate than a mutator class.
It is also much more difficult to expand. 
Is the recommendation to subclass to add shapes?
Is "LargeIcon" really a shape?